scripts/example_script.py

```diff
@@ -0,0 +1,241 @@
+import girder_client
+from girder_client import HttpError
+import json
+
+
+def print_http_error(err):
+    print ('Request Error:\nstatus:%s\nresponse%s\nurl:%s'
+           '\nmethod:%s ' % (err.status, err.responseText, err.url, err.method))
+
+
+def recurse_get_resource(client, parent_folder, resource_type,
+                         resource_list, parent_is_collection=False):
```

Comment:
For consistency with the rest of Girder, how about make `parent_is_collection` into `parentType`, and let it contain either `'collection'` or `'folder'`?